group_vars/mq.yml

```diff
@@ -52,7 +52,7 @@ rabbitmq_users:
   - user: flower
     password: "{{ rabbitmq_password_flower }}"
     tags: administrator
-    vhost: /
+    vhost: /,galaxy
```

Comment:
the `,` looks strange

Reply:
yes, you're right, it's not possible to set multiple vhosts in one `rabbitmqctl set_permissions` command